api/all/src/main/java/io/opentelemetry/api/metrics/HistogramAdviceConfigurer.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.metrics;
+
+import java.util.List;
+
+/** Configure advice for implementations of {@link LongHistogram} and {@link DoubleHistogram}. */
+public interface HistogramAdviceConfigurer {
```

Comment:
do you envision this having more methods added, or could we add `@FunctionalInterface` to it?